lib/nerves_hub_web/components/device_page/logs_tab.ex

```diff
@@ -99,23 +124,55 @@ defmodule NervesHubWeb.Components.DevicePage.LogsTab do
 
   def render(assigns) do
     ~H"""
-    <div class="size-full p-12">
+    <div class="size-full bg-[#0e1019] pb-10">
       <div :if={!@has_logs} class="size-full flex justify-center items-center p-6 gap-6 text-medium font-mono">
         <div>No logs have been received yet.</div>
       </div>
-      <div :if={@has_logs} class="text-base italic font-medium pb-6">Showing the last 25 log lines.</div>
-      <div :if={@has_logs} id="log_lines" phx-update="stream" class="flex flex-col size-full items-start gap-2">
-        <div :for={{dom_id, line} <- @streams.log_lines} id={dom_id} phx-mounted={@log_inserted && fade_in()} class="flex flex-row gap-4 font-mono text-sm">
-          <div id={"#{DateTime.to_unix(line.timestamp, :microsecond)}-log-line-localtime"} phx-hook="LogLineLocalTime">
-            {line.timestamp}
+      <div :if={@has_logs} class="size-full">
+        <div class="flex flex-row items-center justify-between h-11 border-b border-zinc-700 px-12">
+          <div>
+            <span class="text-sm text-zinc-400">Live log streaming :</span>
+            <button
+              id="toggle-log-streaming"
+              type="button"
+              phx-click="toggle_log_streaming"
```

Comment:
We tend to use hyphens for `phx-click` attribute values.

```suggestion
              phx-click="toggle-log-streaming"
```

Reply:
true true. I used underscores initially because I had it as an atom, but then realized that it should be a string, but forgot to switch it back around